src/core/dbus-execute.c

```diff
@@ -1826,6 +1862,33 @@ int bus_exec_context_set_transient_property(
         if (streq(name, "LogRateLimitBurst"))
                 return bus_set_transient_unsigned(u, name, &c->log_ratelimit_burst, message, flags, error);
 
+        if (streq(name, "LogFilterPatterns")) {
+                char *pattern;
+                bool is_allowlist;
+
+                r = sd_bus_message_enter_container(message, 'a', "(bs)");
+                if (r < 0)
+                        return r;
+
+                while ((r = sd_bus_message_read(message, "(bs)", &is_allowlist, &pattern)) > 0) {
+                        r = exec_store_log_filter_pattern(c, pattern, is_allowlist);
```

Comment:
Please do not update the context `c` here, but only check the patterns and store them temporary.